cluster/ci/config/mirroring/codeready-toolchain/codeready_toolchain_v0_1_quay

```diff
@@ -0,0 +1,2 @@
+registry.svc.ci.openshift.org/codeready-toolchain/host-operator-v0.1:host-operator quay.io/codeready-toolchain/host-operator:v0.1
```

Comment:
I'm sorry I missed this in the first review - this file is not needed in this location anymore (it's present  in `core-services/...`).

Reply:
addressed in  fe3f088